docker-compose.yml

```diff
@@ -35,6 +43,11 @@ services:
       - http.host=0.0.0.0
       - transport.host=127.0.0.1
       - xpack.security.enabled=false
+    healthcheck:
+      test: ["CMD", "curl --silent --fail localhost:9200/_cluster/health || exit 1"]
+      interval: 30s
+      timeout: 30s
+      retries: 3
```

Comment:
These seem somewhat magical. Can you include in commit or similar some info on why these values were chosen and why this case needs them and others are ok with the defaults?